src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -179,14 +185,18 @@ public function supportsDenormalization($data, $type, $format = null)
      */
     private function getNormalizer($data, $format)
     {
-        $class = get_class($data);
-        if (isset($this->normalizerCache[$class][$format])) {
-            return $this->normalizerCache[$class][$format];
+        if ($isObject = is_object($data)) {
```

Comment:
if you can pass something else than an object to this method, you need to update the phpdoc